core/cog-shell.h

```diff
@@ -15,6 +15,13 @@
 
 G_BEGIN_DECLS
 
+#define COG_SHELL_ERROR (cog_shell_error_quark ())
+
+typedef enum
+{
```

Comment:
The opening brace goes next to the `typedef enum` in the Cog code style, let's move it up :arrow_up: